editor/utils/test/mobile.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Internal dependencies
+ */
+import { disableIsSidebarOpenedOnMobile } from '../mobile';
+
+describe( 'disableIsSidebarOpenOnMobile()', () => {
+	it( 'should disable isSidebarOpen on mobile and keep other properties as before', () => {
+		const input = {
+			isSidebarOpened: true,
+			dummyPref: 'dummy',
+		};
+		const output = {
+			isSidebarOpened: false,
+			dummyPref: 'dummy',
+		};
+		expect( disableIsSidebarOpenedOnMobile( input, true ) ).toEqual( output );
+	} );
+
+	it( 'should keep isSidebarOpen on non-mobile and keep other properties as before', () => {
+		const input = {
+			isSidebarOpened: true,
+			dummy: 'non-dummy',
+		};
+		const output = {
+			isSidebarOpened: true,
+			dummy: 'non-dummy',
+		};
+		expect( disableIsSidebarOpenedOnMobile( input, false ) ).toEqual( output );
+	} );
```

Comment:
Tiny suggestion: right now, the memory is fresh and the signature of `disableIsSidebarOpenedOnMobile` is known. Would it be clearer to add a variable like so:

```js
const input = …, output = …, isMobile = true;
expect( disableIsSidebarOpenedOnMobile( input, isMobile ) … 
```

?